src/Intervals.jl

```diff
@@ -57,7 +57,6 @@ export Bound,
        union!,
        less_than_disjoint,
        greater_than_disjoint,
-       isdisjoint,
```

Comment:
was this just left over from some other change previously?

Reply:
This was added to Base in Julia 1.5, so is now always available and doesn't need to be exported
https://docs.julialang.org/en/v1/base/collections/#Base.isdisjoint

Reply:
Should have been removed as part of: https://github.com/invenia/Intervals.jl/pull/186